src/nbl/asset/interchange/CPLYMeshFileLoader.cpp

```diff
@@ -24,7 +22,11 @@ namespace nbl
 namespace asset
 {
 
-CPLYMeshFileLoader::CPLYMeshFileLoader(IAssetManager* _am) : IRenderpassIndependentPipelineLoader(_am) {}
+CPLYMeshFileLoader::CPLYMeshFileLoader(IAssetManager* _am) 
+	: IRenderpassIndependentPipelineLoader(_am)
+{
+	initialize();
```

Comment:
sop calling `initialize` in the constructor

Reply:
corrected in following commit